### PR TITLE
Clarify the conventions used by the CI system

### PR DESCRIPTION
**Pull request by gber**
Followup to https://github.com/ubports/docs.ubports.com/pull/480#pullrequestreview-857050201 .

Please do not merge until reviewed by @peat-psuwit

### CODE REVIEW

File: about/process/branch-naming.rst [hunk 1]
```diff
@@ -17,11 +17,15 @@ To make most efficient use of our CI system, a special naming convention for Git
...
 
-Usually there will be branches corresponding to each release of Ubuntu Touch which will be used for stable release updates whereas development takes place in the ``main`` branch. If no separate versions for the different Ubuntu bases are needed, the repository will have one ``main`` branch.
-In the presence of release-specific branches the CI system will only build packages for the specific release a branch is associated with by name or the development version in case of the ``main`` branch. If e.g. a branch ``ubports/xenial`` exists and a commit is added to it, packages will only be built for Ubuntu Touch 16.04 but not any other release.
-If only a ``main`` branch exists, packages will be built for all actively supported releases.
+The CI system automatically builds packages for the aforementioned branches for different releases based on the following conventions:
```
Comment (peat-psuwit) [inline about/process/branch-naming.rst:22]: I guess

```suggestion
The CI system automatically builds packages for the aforementioned branches for different releases based on the following rules:
```

works better.

File: about/process/branch-naming.rst [hunk 2]
```diff
@@ -17,11 +17,15 @@ To make most efficient use of our CI system, a special naming convention for Git
...
+
+If a package is not part of a released version of Ubuntu Touch yet, only a ``main`` branch is needed.  Packages will be built for the current in-development version as well as all supported stable releases.  If the latter is not desired, branches containing only a README file should be created for those releases.
+
+If a package is part of a released version of Ubuntu Touch, the ``main`` branch will be built for the current in-development version as well as supported stable versions for which no release-specific branch exist.  Release-specific branches will only be built for that release.  If e.g. a branch ``ubports/xenial`` exists and a commit is added to it, packages will only be built for Ubuntu Touch 16.04 but not any other release.
```
Comment (peat-psuwit) [inline about/process/branch-naming.rst:26]: Maybe make these bullet points?

File: about/process/branch-naming.rst [hunk 3]
```diff
@@ -17,11 +17,15 @@ To make most efficient use of our CI system, a special naming convention for Git
...
+
+Packages built from these branches are directly built into their respective images and also published in the package repositories on ``repo.ubports.com`` for stable releases and ``repo2.ubports.com`` in case of the current development version.
```
Comment (peat-psuwit) [inline about/process/branch-naming.rst:28]: The plan is to abandon repo.ubports.com in the future and continue using repo2 even after Focal becomes stable. So, this is not quite accurate.

**Review by peat-psuwit — CHANGES_REQUESTED** [review]

File: about/process/branch-naming.rst (hunk 3 above)
Comment (peat-psuwit) [reply about/process/branch-naming.rst:28]: (BTW, repo2 is using [Aptly](https://github.com/aptly-dev/aptly), while repo is using [Freight](https://github.com/rcrowley/freight).)

File: about/process/branch-naming.rst (hunk 2 above)
Comment (gber) [reply about/process/branch-naming.rst]: OK.

File: about/process/branch-naming.rst (hunk 3 above)
Comment (gber) [reply about/process/branch-naming.rst]: > The plan is to abandon repo.ubports.com in the future and continue using repo2 even after Focal becomes stable. So, this is not quite accurate.

OK, I've corrected that.

> (BTW, repo2 is using [Aptly](https://github.com/aptly-dev/aptly), while repo is using [Freight](https://github.com/rcrowley/freight).)

Does that need to be mentioned? Target audience here are developers/contributors doing packaging work.

Comment (peat-psuwit) [reply about/process/branch-naming.rst]: > Does that need to be mentioned?

Not really. Just mention for completeness.

**Review by peat-psuwit — APPROVED** [review]
It sounds better now. Thank you.

@doniks, do you prefer me to merge something like this or you do it yourself?

Also, how should we document on selecting branch to contribute and our back-merge policy? (It was made official that we back-merge `ubports/xenial` to `main` every now and then in a devsync some time ago.)

**Comment by doniks** [comment]
> @doniks, do you prefer me to merge something like this or you do it yourself?

If you checked it and approve, I think it's best you also merge it. In case you would *want* me to check smt, do let me know.

**Pull request merged**